Scrabble.java

```diff
@@ -48,24 +51,43 @@ public static void init() {
 
 	// Checks if the given word is in the dictionary.
 	public static boolean isWordInDictionary(String word) {
-		//// Replace the following statement with your code
+		for(int i = 0;i < DICTIONARY.length;i++){
+			if (word == DICTIONARY[i]) {
+				return true;
+			}
+		}
 		return false;

```

Comment:
1. run up to NUM_OF_WORDS
2. use string.equals.. example: `if (word.equals(DICTIONARY[i]))`